docs/configuration/configuration.md

```diff
@@ -31,7 +31,7 @@ An example [oauth2_proxy.cfg]({{ site.gitweb }}/contrib/oauth2_proxy.cfg.example
 | `-client-secret` | string | the OAuth Client Secret | |
 | `-client-secret-file` | string | the file with OAuth Client Secret | |
 | `-config` | string | path to config file | |
-| `-cookie-domain` | string | an optional cookie domain to force cookies to (ie: `.yourcompany.com`) | |
+| `-cookie-domain` | string \| list | Optional cookie domains to force cookies to (ie: `.yourcompany.com`). The longest domain matching the request's host will be used (or the shortest cookie domain if there is no match). | |
```

Comment:
```suggestion
| `-cookie-domain` | string | list | Optional cookie domains to force cookies to (ie: `.yourcompany.com`). The longest domain matching the request's host will be used (or the shortest cookie domain if there is no match). | |
```

There's an unneeded backslash here.

Reply:
I think the backslash is actually required, it shows the pipe character in the table so you can see this is a list of strings, we've done it for other options in the table before

Reply:
I believe this backslash is required. I used `whitelist-domain` as a reference, which uses the backslash in order to indicate both string & list in the "Type" column:
<img width="758" alt="image" src="https://user-images.githubusercontent.com/6352776/75590026-3bb77e80-5a31-11ea-893d-9bf42388f093.png">